src/command/output.rs

```diff
@@ -107,6 +111,27 @@ impl RoverStdout {
                 print_descriptor("Introspection Response");
                 println!("{}", &introspection_response);
             }
+            RoverStdout::Markdown(markdown_string) => {
+                // underline bolded md
+                let mut skin = MadSkin::default();
+                skin.bold.add_attr(Underlined);
+
+                // replace links in format `[this](url)` to `this (url)`, since
+                // termimad doesn't handle links for us.
+
+                // this pattern captures the named groups, <title> and <url_with_parens>
+                // that we can use to replace with later
+                let re = Regex::new(r"\[(?P<title>[^\[]+)\](?P<url_with_parens>\(.*\))").unwrap();
```

Comment:
i like the formatting here and we can def keep things the way they are if we find them useful, but it _might_ be a tad easier to just format the links like this in the markdown files themselves instead of real markdown links? 

that might bite us if we ever want to integrate these error docs into our netfliy docs directly though.... something to think about

Reply:
> that might bite us if we ever want to integrate these error docs into our netfliy docs directly though

Yeah that was my thinking. I think it'd be nice to either do it automatically or have a step to cargo `rover explain EALL` and copy into a docs error "index" of sorts as part of the release process

I also haven't looked, but there may be a better way to manipulate it, with some specific markdown transformer/parser somewhere. But I think this could work for now. Heck, even if we didn't replace the links at all, I think having a MD link in the terminal isn't the _worst_ thing in the world, since it's still clickable (that's what rustc does)

Reply:
mmmm! yeah honestly if it's good enough for rustc it's good enough for me! i vote we just print it as regular md links, but it's like a 51/49 situation, I'm really OK with either.